src/code.l

```diff
@@ -2566,7 +2566,8 @@ RAWEND    ")"[^ \t\(\)\\]{0,16}\"
 					  generateClassOrGlobalLink(*g_code,yytext);
   					  g_name+=yytext; 
 					}
-<Body>{SCOPENAME}/{BN}*[:;,)\]]		{ // "int var;" or "var, var2" or "debug(f) macro" , or int var : 5;
+<Body>{SCOPENAME}/{BN}*[:;,)\[]		{ // "int var;" or "var, var2" or "debug(f) macro" , or "int var : 5;"
```

Comment:
You removed `]` from the pattern, but I think this breaks highting symbolic constant `b` in this code `a[b]`